README.md

```diff
@@ -36,6 +36,24 @@ python netbox/manage.py dumpdata \
 
 From within the Nautobot application environment, run `nautobot-server import_netbox_json <json_file> <netbox_version>`, for example `nautobot-server import_netbox_json /tmp/netbox_data.json 2.10.3`.
 
+### Import ChangeLog/ObjectChange records
+
+Because the ChangeLog can be a massive amount of data and sometimes the historic would be omitted, the import of the `ObjectChange` is omitted in the default import command. To import it, **after** the previous Netbox import process has succeeded.
```

Comment:
```suggestion
Because the database change log can be a massive amount of data, and often this historical information does not need to be imported, `ObjectChange` records are not included in the database export command above and are not handled by the `import_netbox_json` command. To import `ObjectChange` records specifically, **after** the previous Netbox import process has succeeded, you can do the following.
```